Source/WebCore/css/CSSProperties.json

```diff
@@ -9426,6 +9426,71 @@
                 "url": "https://drafts.csswg.org/css-anchor-position-1/#position-anchor"
             }
         },
+        "position-area": {
+            "values": [
+                "block-end",
```

Comment:
If we are parsing the value ourselves, I don't know if this would do anything. I think it also gives the wrong impression that these are the values that `position-area` accepts, while the reality is more complicated.

Reply:
It mostly lists them on webkit.org/css-status , but nothing else really.

(Also `none` is missing from this list)

I'm fine either way regarding keeping this list.